.travis.yml

```diff
@@ -1,5 +1,3 @@
 language: ruby
 rvm:
- - 2.1.2
- - 2.1.8
- - 2.2.4
+ - 2.3.0
```

Comment:
Yep.  Since upstream has diverged a fair bit (with that giant refactor PR they had), I don't expect to get these changes merged.  

Some of the code I'm using requires later versions.  I could write it to use older versions, but in the interest of GSD I'm just making it work.

Later on I want to give some thought to what we should do long-term about this project and its divergence from upstream.